input/config/aarch64/rhel/10.0/slurm_custom.json

```diff
@@ -2,8 +2,11 @@
     "slurm_custom": {
         "cluster": [
             {"package": "munge", "type": "rpm", "repo_name": "aarch64_appstream"},
+            {"package": "munge-devel", "type": "rpm", "repo_name": "aarch64_codeready-builder"},
```

Comment:
@balajikumaran-c-s Move this to openmpi.json. Check if this is required for slurm?